core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2009,7 +2010,7 @@ private void renameInternal(RpcContext rpcContext, LockedInodePath srcInodePath,
       mPersistRequests.put(srcInode.getId(), new alluxio.time.ExponentialTimer(
           ServerConfiguration.getMs(PropertyKey.MASTER_PERSISTENCE_INITIAL_INTERVAL_MS),
           ServerConfiguration.getMs(PropertyKey.MASTER_PERSISTENCE_MAX_INTERVAL_MS),
-          ServerConfiguration.getMs(PropertyKey.MASTER_PERSISTENCE_INITIAL_WAIT_TIME_MS),
+          srcInode.asFile().getPersistenceWaitTime(),
```

Comment:
This should not obey the file's persist wait time.

For the persist on rename feature, we should expect users to set a persist wait time of `-1` (or some other special value) to indicate that the file will be persisted but should not be automatically persisted by Alluxio (rather through a rename or CLI persist).

Reply:
In the rename context, we have a `context.getPersist`. Only if `Context.getPersist (a persist operation should follow the rename)`, will we trigger the rename. Then here it should also obey the `srcInode.getShouldPersistTime`. 

If users don't want the file be automatically persisted by Alluxio, then they didn't need to pass in the `Context.getPersist` and directly trigger `persist CLI` when they need. Looks like we don't need the special persist wait time special value.